.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: Test final Docker image
       run: |
         docker run --rm docker.pkg.github.com/lifting-bits/mcsema/mcsema-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest --version
-        docker run --rm --entrypoint=mcsema-disass docker.pkg.github.com/lifting-bits/mcsema/mcsema-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest --help
+        docker run --rm --entrypoint=mcsema-disass-2 docker.pkg.github.com/lifting-bits/mcsema/mcsema-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest --help
```

Comment:
Do we only check python2 since it uses dyninst?

Reply:
We only check python2 because Python3 is not installed in the `dist` image.

There is an issue with the `PYTHONPATH` variable if we intend to have both Python2 and Python3 working from the Docker image, since it currently points to the Python2 libs